bootstrap/auto-updates.sls

```diff
@@ -0,0 +1,24 @@
+---
+
+unattended-upgrades:
+  pkg:
+    - installed
+
+/etc/apt/apt.conf.d/50unattended-upgrades:
+  file:
+    - managed
+    - source: salt://bootstrap/templates/unattended-upgrades
+    - mode: 440
+    - template: jinja
+    - require:
+      - pkg: unattended-upgrades
+
+/etc/apt/apt.conf.d/20auto-upgrades:
```

Comment:
Why are we doing automatic upgrades?

Reply:
required by the apt cron to install them, see annotated file

Reply:
see comments here https://wiki.debian.org/UnattendedUpgrades